pkg/cmd/server/origin/master_config.go

```diff
@@ -223,6 +231,7 @@ func BuildMasterConfig(options configapi.MasterConfig) (*MasterConfig, error) {
 	const defaultInformerResyncPeriod = 10 * time.Minute
 	internalkubeInformerFactory := kinternalinformers.NewSharedInformerFactory(privilegedLoopbackKubeClientsetInternal, defaultInformerResyncPeriod)
 	externalkubeInformerFactory := kinformers.NewSharedInformerFactory(privilegedLoopbackKubeClientsetExternal, defaultInformerResyncPeriod)
+	authorizationInformers := authorizationinformer.NewSharedInformerFactory(authorizationClient, 5*time.Minute)
```

Comment:
Can you please add a TODO here, we should have a single place where we'll start all the informers (like k8s has), I've already opened https://github.com/openshift/origin/issues/14108 to address the switch.